rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -330,6 +330,14 @@ RCLCPP_PUBLIC
 std::vector<const char *>
 get_c_vector_string(const std::vector<std::string> & strings_in);
 
+/// Call dds to notify about new network interfaces.
```

Comment:
This layer shouldn't assume that the underlying middleware is a DDS.
Maybe the comment should say something like "Notify middleware to refresh available network interfaces?"

Reply:
@alsora good point - will change that.